src/sql/command.py

```diff
@@ -24,7 +25,9 @@ def __init__(self, magic, user_ns, line, cell) -> None:
         self._line = line
         self._cell = cell
 
-        self.args = parse.magic_args(magic.execute, line)
+        ALLOWED_DUPLICATES = ["-w", "--with", "--append", "--interact"]
+        self.args = parse.magic_args(magic.execute, line, "sql", ALLOWED_DUPLICATES)
```

Comment:
`self.args = parse.magic_args(magic.execute, line, "sql",  allowed_duplicates=["-w", "--with", "--append", "--interact"])`
and removed the `ALLOWED_DUPLICATES` variable

Reply:
Fixed [here](https://github.com/ploomber/jupysql/pull/908/files#diff-769097f5e36be57ddec10c78df0900db8c1e254f5485bc94367aa7fcb07991c5R29-R34)